bin/apisix

```diff
@@ -44,6 +45,16 @@ else
 end
 -- print("apisix_home: ", apisix_home)
 
+do
+    local ok, json = pcall(require, "cjson")
```

Comment:
We should specify in commit why we changed it, otherwise, other developers don't know why.

The reason for this modification is that if the user installs cjson using luarocks first and in the environment of lua 5.3, there will be problems when using apisix. 

Because apisix uses OpenResty, which only supports cjson compiled by luajit and lua5.1. cjson is unstable in other environments.